docs/spec-files/title.md

```diff
@@ -6,9 +6,9 @@ DIDComm Messaging
 **Latest published version:**
   [identity.foundation/didcomm-messaging/spec](https://identity.foundation/didcomm-messaging/spec)
 
-**Editors:**
+**Editors:** Sam Curren (Indicio), Tobias Looker (Mattr), Oliver Terbu (ConsenSys)
 
-**Contributors:** Daniel Hardman
+**Contributors:** Sam Curren (Indicio), Tobias Looker (Mattr), Oliver Terbu (ConsenSys), Kyle den Hartog (Mattr), Baha Shaaban (SecureKey), Drummond Reed (Evernym), Steve McCown (Anonyme Labs), Troy Ronda (SecureKey), George Aristy (SecureKey), Vyacheslav Gudkov (DSR), Alexander Shcherbakov (DSR), Alexander Martynov (DSR), Daniel Buchener (Microsoft), Devin Fisher (Evernym), Orie Steele (Transmute), Brian Richter (?), @bumblefudge, @liormargalit, Timo Glastra (Animo Solutions), Andrew Whitehead (Goverment of British Columbia), Nader Helmy (?), Markus Sabadello (Danube Tech), Patrick McClurg (SICPA), Stephen Curran (?), Daniel Hardman (SICPA)
```

Comment:
```suggestion
**Contributors:** Sam Curren (Indicio), Tobias Looker (MATTR), Oliver Terbu (ConsenSys Mesh), Kyle Den Hartog (MATTR), Baha Shaaban (SecureKey), Drummond Reed (Evernym), Steve McCown (Anonyome Labs), Troy Ronda (SecureKey), George Aristy (SecureKey), Vyacheslav Gudkov (DSR), Alexander Shcherbakov (DSR), Alexander Martynov (DSR), Daniel Buchner (Microsoft), Devin Fisher (Evernym), Orie Steele (Transmute), Brian Richter (?), Juan Caballero (Spruce), @liormargalit, Timo Glastra (Animo Solutions), Andrew Whitehead (Government of British Columbia), Nader Helmy (MATTR), Markus Sabadello (Danube Tech), Patrick McClurg (SICPA), Stephen Curran (?), Daniel Hardman (SICPA)
```

Reply:
Sorry -- I can't see that I can edit this.  For my "?", please change to "Government of British Columbia".

Thanks

Reply:
Updated.